worlds/doom_1993/Options.py

```diff
@@ -0,0 +1,84 @@
+import typing
+
+from Options import AssembleOptions, Choice, Toggle, DeathLink, DefaultOnToggle
+
+
+class Difficulty(Choice):
+    """
+    Choose the difficulty option. Those match DOOM's difficulty options.
+    baby (I'm too young to die.) double ammos, half damage, less monsters or strength.
+    easy (Hey, not too rough.) less monsters or strength.
+    medium (Hurt me plenty.) Default.
+    hard (Ultra-Violence.) More monsters or strength.
+    nightmare (Nightmare!) Monsters attack more rapidly and respawn.
+    """
+    display_name = "Difficulty"
+    option_baby = 0
+    option_easy = 1
+    option_medium = 2
+    option_hard = 3
+    option_nightmare = 4
+    default = 2
+
+
```

Comment:
Have you considered having a completely random monsters option? Or would that just straight up not work?

Reply:
Yeah that's fair lol.

Reply:
Could add a slider tho. how much % to respect that "Ratio". From 0 to 100. 100 = respect ratio, 0 = fully random.

Reply:
I'm just here for the chaos of it, and it's always so much fun to see the screenshots or stories from people who decide to set it to fully random.

Reply:
I'll not it for a future release, but won't do that immediately as it needs more balancing/work.

Reply:
https://trello.com/c/lDALpnzG/157-add-slider-for-random-monster-from-balanced-fully-random